### ISSUE TITLE
`htu` claim is not properly constructed when there are tiple slashes in the URL. 

### ISSUE THREAD

**Issue by angelaraya** [inrupt/solid-client-authn-js#151]
**Description of the bug**
When  requesting a private resource located at `https://{lpd:storage}/private.ttl` , the DPoP HTTP header contains the claim `"htu": "null/{ldp:storage}/private.ttl"`. From what I can tell that `null` should be `https`. This is making the resource server respond with a `403 Forbidden`.

**Desktop information**
 - OS: OS 10.15.5
 - Browser: Firefox 77.0.1

**Comment by jaxoncreed** [comment]
What does your "login" call look like? And specifically how are you making the "fetch" call?

**Comment by angel-araya-jd** [comment]
```ts
const options = const authOptions = {
  oidcIssuer: 'https://oidc.provider.tld/',
  redirect: 'http://localhost:8080',
  clientId: '----'
} as ILoginInputOptions;

await login(options);
```
```ts
const storage = getStorage(this.webId);
const r = await fetch(`${storage}/private.ttl`, {
    headers: {
        accept: 'text/turtle'
   }
});
```

**Comment by jaxoncreed** [comment]
What is the value of "storage"?

**Comment by angel-araya-jd** [comment]
The value is the URL for `ldp:storage` in the webId. I checked and the URL is valid and the `private.ttl` document exists in there.

**Comment by angel-araya-jd** [comment]
I found the problem. It was as you probably suspected a problem with the path. Somewhere along the our implementation a triple slash was added making the URL making it `https:///ldp...`. In all other environments I tried (Firefox's `fetch`, `axios`) the request succeeded regardless of the number of slashes so I'd still consider it a bug. It should resolve it in the same manner as the other HTTP clients.
